src/test/kotlin/no/nav/tilleggsstonader/sak/vilkår/VilkårServiceTest.kt

```diff
@@ -51,7 +56,12 @@ internal class VilkårServiceTest {
         fagsakService = fagsakService,
     )
     private val søknad = SøknadsskjemaMapper.map(
-        SøknadUtil.søknadskjemaBarnetilsyn(),
+        SøknadUtil.søknadskjemaBarnetilsyn(
+            barnMedBarnepass = listOf(
+                barnMedBarnepass(ident = "19852098840"),
+                barnMedBarnepass(ident = "13811397592"),
```

Comment:
For at testet ikke skal feile etter X antall år så skulle vi kunne bruke FnrGenerator som genererer et random-fnr for ett gitt dato
```suggestion
                barnMedBarnepass(ident = FnrGenerator.generer(Year.now().minusYears(1).value, 5, 19)),
                barnMedBarnepass(ident = FnrGenerator.generer(Year.now().minusYears(11).value, 1, 13)),
```